#
#

Header header
int32 maxSpeed # Max speed, 0 means doesn't change.
int32 accTime  # Accelerate time 1-32767ms, 0 means doesn't change.
int32 decTime  # Decelerate time 1-32767ms, 0 means doesn't change.
